### ISSUE TITLE
Cross-platform communications support with twemoji

### ISSUE THREAD

**Issue by doelgonzo** [joeattardi/emoji-button#162]
For environments that don't have native emoji support (i.e. Windows7), we don't have a way to "translate" emojis to their `twemoji` counterparts. Would it be at all possible to have some flag to expose the `twemoji` counterpart when **not** using `twemoji`? We want to avoid using `twemoji` as the global picker, as the native experience tends to be better anyway, moreover we wouldn't want to make the UI unfamiliar to non-Windows users.

**Usecase:** A messaging app with one user on iOS sending an emoji to another in Windows7. Every other client will see the emoji, but the Windows user will have a box glyph. If we could expose the `twemoji` URL, we could have logic that detects "is windows" and presents the URL from `twemoji` instead of the native emoji. 

Basically this alternative would be to allow the selection component to utilize the native picker and have it always spit out the `twemoji` URL-equivalent so we can handle environments where the emoji isn't available.

**Comment by joeattardi** [comment]
I'm not a fan of mixing twemoji and native emojis in a single picker. Might not even be possible with the new design I'm working on for the renderers in v5.

If you want to use twemoji to fill in the gaps, why are you opposed to using twemoji as a whole?

**Comment by doelgonzo** [comment]
My suggestion was more along the lines of "if there is a `twemoji` for XYZ emoji, include the url attribute". I'm not saying the picker itself has to show it for the missing case, instead expose the information and let the developer consuming the emoji decide which one to show.

Right now I have implemented that if the OS is Linux/Windows, to show the twemoji picker, but if someone sent an emoji from iOS (which doesnt use `twemoji`), it has nothing it can show a Linux user without the font installed. It's about exposing the information, not making the picker UI different in any way
